db/migrate/20221102135740_add_devise_to_users.rb

```diff
@@ -4,28 +4,28 @@ class AddDeviseToUsers < ActiveRecord::Migration[7.0]
   def self.up
     change_table :users do |t|
       ## Database authenticatable
-      t.string :email,              null: false, default: ""
-      t.string :encrypted_password, null: false, default: ""
+      #t.string :email,              null: false, default: ""
+      #t.string :encrypted_password, null: false, default: ""
```

Comment:
- [ ] Your app produces an error because you don't have the email in your user's table, kindly uncomment this code (email) so your views render correctly

![Screenshot_20221102_191347](https://user-images.githubusercontent.com/25227022/199571093-40feb815-ef53-4fc1-9edc-87ac1f1d2b2e.png)

Reply:
Many thanks for the comment.
Resolved